packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -93,6 +119,27 @@ const getDocs: () => JSX.Element = () => {
       <Canvas mdxSource={FloatingLocalVideoExampleText}>
         <FloatingLocalVideoExample />
       </Canvas>
+      <Subheading>Speaker layout</Subheading>
+      <SingleLineBetaBanner></SingleLineBetaBanner>
+      <Description>
+        This Layout is meant to highlight the current dominant speaker in the call. For this view in the video gallery
```

Comment:
nit: For new paragraphs/sections, avoid `this`
```suggestion
        Speaker Layout is meant to highlight the current dominant speaker in the call. For this view in the video gallery
```